.github/workflows/upload-go-master.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - env:
           CGO_ENABLED: 0
         run: |
-          go build -ldflags "-X github.com/SAP/jenkins-library/cmd.GitCommit=${GITHUB_SHA} \
+          go build -ldflags "-w -s -X github.com/SAP/jenkins-library/cmd.GitCommit=${GITHUB_SHA} \
```

Comment:
Could you add a docs link in here? In 2 months I don't remember what this means of where to look it up.

Reply:
Maybe this works:

```suggestion
          go build -ldflags "-w \ #...
          -s \ # omit debug infos
          -X github.com/SAP/jenkins-library/cmd.GitCommit=${GITHUB_SHA} \
```

Reply:
Yes, sadly there is no long options which I would prefer. I updated with a comment that should make it clear.